src/mito2/Cargo.toml

```diff
@@ -14,7 +14,6 @@ workspace = true
 
 [dependencies]
 api.workspace = true
-aquamarine.workspace = true
```

Comment:
We can't remove aquamarine because the document relies on this crate. Could you ignore the usage of this crate in all `Cargo.toml`?
https://github.com/mersinvald/aquamarine